src/chigame/api/filters.py

```diff
@@ -0,0 +1,13 @@
+import django_filters
+
+from chigame.games.models import Game
+
+
+class GameFilter(django_filters.FilterSet):
+    name = django_filters.CharFilter(lookup_expr="icontains")
+    min_players = django_filters.NumberFilter(lookup_expr="exact")
+    max_players = django_filters.NumberFilter(lookup_expr="exact")
+
+    class Meta:
+        model = Game
+        fields = ["name", "min_players", "max_players"]
```

Comment:
With the merge of the new game model into dev, this will need to be updated to include the new fields. This can be in a new issue, and for now the game endpoint and filtering will still work, even if not displaying all of the info for the game.

Reply:
I will create a new issue to update the filtering fields for the games.